prow/cmd/README.md

```diff
@@ -39,9 +48,30 @@ These are small tools that are automatically added to ProwJob pods for jobs that
 
 * [`clonerefs`](/prow/cmd/clonerefs)
 * [`initupload`](/prow/cmd/initupload)
+* [`gcsupload`](/prow/cmd/gcsupload)
```

Comment:
I don't think this is part of pod utilities

Reply:
Where should this go instead?

Reply:
not quite sure, let me take a look at what it is and where it's used

Reply:
`cmd/gcsupload` is not referenced anywhere other than https://github.com/kubernetes/test-infra/tree/master/prow/cmd/sidecar#sidecar, mentioning a similarity between gcsupload and sidecar. The owner of gcsupload https://github.com/kubernetes/test-infra/blob/master/prow/cmd/gcsupload/OWNERS is @stevekuznetsov themselves, so they might know more?

Reply:
I'm going to move to undocumented for now